src/main/java/spireMapOverhaul/zones/beastslair/BeastsLairEvent.java

```diff
@@ -0,0 +1,78 @@
+package spireMapOverhaul.zones.beastslair;
+
+import basemod.AutoAdd;
+import basemod.ReflectionHacks;
+import basemod.abstracts.events.PhasedEvent;
+import basemod.abstracts.events.phases.CombatPhase;
+import basemod.abstracts.events.phases.TextPhase;
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.SpawnMonsterAction;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.EventStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.ArtifactPower;
+import javassist.CtBehavior;
+import spireMapOverhaul.actions.AllEnemyApplyPowerAction;
+import spireMapOverhaul.util.Wiz;
+import spireMapOverhaul.zones.beastslair.powers.FuryPower;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+@AutoAdd.Ignore
```

Comment:
I don't think you need this. When the event is part of a zone, it won't show up unless you specifically set it to do so with the `ModifiedEventRateZone` interface or a forced event like this zone uses.

The issue with the example event was because it wasn't part of a zone anymore, so it got registered as a generic event that could show up anywhere.